smart-on-fhir/0-introduction/introduction.md

```diff
@@ -1,45 +1,34 @@
 # Introduction
 
-## About this Workshop
+## Workshop Overview
 
-This introduction covers the complete "parent" workshop. Use this text to set up the story for the workshop. Be engaging - what will the learner get from spending their time on this workshop?
+Welcome to the Oracle LiveLabs Workshop: Building an ASCVD Risk Calculator! In this hands-on session, we will guide you through the process of creating a powerful application using Oracle Application Express (APEX) and the Code Console. By the end of this workshop, you will have a fully functional ASCVD (Atherosclerotic Cardiovascular Disease) Risk Calculator at your disposal.
```

Comment:
"using Oracle Application Express (APEX) and the Code Console" change it in "using Oracle Application Express (APEX) and  FHIR API's"

Reply:
Updated it, thanks!